### DISCUSSION TITLE
Destroy(Object obj, float t = 0.0F); 사용이 불가능한가요?

### DISCUSSION THREAD

**Discussion by bestdeveloperjo** [naverz/zepeto-studio-kor#996] [5. Q&A]
export default class NewTypescript extends ZepetoScriptBehaviour {
    public speed = 8;
    private  bulletRigidbody : Rigidbody;
    eh = 0;

    Start() {    
     this.bulletRigidbody = this.GetComponent<Rigidbody>();
     this.bulletRigidbody.velocity = this.transform.forward;
     this.eh = this.bulletRigidbody.velocity * this.speed;
     Debug.Log(this.eh);

     Destory(this.gameObject,3);
    }
    
    
}

최대한 C# 문구를 javascript로 바꾸어 적었지만 아래 Destory는 활용이 안되는거 같습니다.
이를 대체할 만한게 있습니까?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, Destory는 다음과 같이 사용할 수 있습니다.
또한 가이드에 관련 내용이 있으니 참고하여 주세요.
https://studio.zepeto.me/kr/guides/gameobjects-and-components

```typescript

import {GameObject, Object} from 'UnityEngine';

public testObject: GameObject;

Start()
{
      GameObject.Destroy(this.gameObject, 10);
}

```
